modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
 
 .. For `degraded`, the broken heart icon is displayed.
-.. For `out of sync`, the yellow yield icon is displayed.
\ No newline at end of file
+.. For `out of sync`, the yellow yield icon is displayed.
+
+. Click on the *Deployment History* tab to view the application deployment history. The page includes details such as, *Last deployment*, *Description* (commit message), *Environment*, *Outcome* (Succeeded/Failed), *Author* and *Revision*.
```

Comment:
Hi @DebarghoGhosh 
Please remove the **Outcome** field. The screenshot you referred to was outdated.